docs/pages/setup/reference/networking.mdx

```diff
@@ -50,12 +50,15 @@ following use cases:
 
 Some networks funnel all connections through a proxy server where they can be
 audited and access control rules can be applied. For these scenarios, Teleport
-supports HTTP CONNECT tunneling.
+supports HTTP CONNECT tunneling. HTTP CONNECT applies to:
+
+- `tsh` in all cases
+- Reverse tunnel agents dialing back to proxies.
```

Comment:
I'm not sure that readers will understand "Reverse tunnel agents" unless they have spent a lot of time in our docs. I would recommend something like, "Teleport services, such as the SSH Service and Database Service, that dial back to the Teleport Proxy Service". Also, both list items should end with the same punctuation.